packages/subgraph/tasks/deploy.sh

```diff
@@ -132,7 +132,7 @@ deploy_to_goldsky() {
     )
 
     local goldskyNetwork="${legacyNetworkNames[$network]:-$network}"
-    local subgraphName="protocol-$DEPLOYMENT_ENV-$goldskyNetwork/$VERSION_LABEL"
+    local subgraphName="protocol-$DEPLOYMENT_ENV-$goldskyNetwork/1.0.0"
```

Comment:
This should be `$VERSION_LABEL`?

Reply:
The version label from the script is the commit SHA. It changes for each deployment and it will lead to different subgraph  version on Goldsky. Being that the version is added to the subgraph url, we'll be forced to change the url for every new deployment if a static version (i.e 1.0.0) is not used.

Reply:
Goldsky supports tagging, that's what we should point everything at, not the concrete versions.